data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+
+mission "Bactrian Unlock [1]: Pirates"
+	invisible
+	landing
+	to offer
+		has "Bactrian Unlock [0]: done"
+	to fail
+		or
+			has "Bactrian Unlock [3]: done"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc kill
+		personality nemesis staying
+		system Arneb
+		government Pirate
+		ship "Firebird (Plasma)" "The Court Jester"
+		ship "Firebird (Plasma)" "The Knighted One"
+		ship "Headhunter (Particle)" "Cerberus"
+		ship "Headhunter (Particle)" "Charon"
+
+mission "Bactrian Unlock [1]: Escorts"
+	invisible
+	landing
+	source Memory
+	to offer
+		has "Bactrian Unlock [1]: active"
+	to fail
+		or
+			has "Bactrian Unlock [3]: offered"
+			has "Bactrian Unlock [3-alt]: offered"
+	
+	npc escort kill
+		government Republic
+		personality heroic escort
+		fleet
+			names "deep"
+			variant
+				"Mule (Heavy)" 2
+				"Aerie"
+				"Dagger" 2
+	
+	on offer
+		conversation
+			`You notice a group of Deep Security warships, two Mules and an Aerie, loading up on ammunition just outside of the spaceport. After thirty minutes they tell you that they are ready to launch and join in on the rescure mission.`
+				accept
+
+mission "Bactrian Unlock [1]: Reinforcements"
+	invisible
+	landing
+	source Farpoint
+	to offer
+		has "Bactrian Unlock [1]: active"
+	to fail
+		or
+			has "Bactrian Unlock [3]: offered"
+			has "Bactrian Unlock [3-alt]: offered"
+	
+	npc escort kill
+		government Republic
+		personality heroic escort
+		fleet
+			names "republic capital"
+			fighters "republic fighter"
+			variant
+				"Cruiser" 2
+				"Combat Drone" 8
+	
+	on offer
+		conversation
+			`While landing on <planet> to refuel, you are approached by two Navy officers who ask if you are involved in the rescue mission. When you respond yes, they inform you that they will be bringing two Cruisers to assist in retrieving the Star Queen.`
+				accept
+
+mission "Bactrian Unlock [2]"
+	landing
+	name "Return to <destination>"
+	description "Now that you have secured the Star Queen, return to <destination> with it intact."
+	destination Valhalla
+	to offer
+		has "Bactrian Unlock [1]: done"
+	on offer
+		conversation
+			`The remaining fleet quickly enters the atmosphere of <planet> and begins scanning the surface. After about an hour of flying, you get a transmission that the ship has been found along with the location of the Star Queen.`
+			`	You watch from afar as the Corvette lands near a hangar in some sort of pirate facility. The Corvette immediately unloads what looks to be at least four dozen armed soldiers who quickly surround and breach the hangar. After about ten minutes, the hangar doors open and out rolls the Star Queen, still intact. You hear chatter over the radio saying that most of the crew and scientists are still inside.`
```

Comment:
maybe replace "rolls" with "hovers", but if it fits lorewise, i'd keep the rolls.